examples/hello_world.cz

```diff
@@ -0,0 +1,5 @@
+with sys
+
+fun hello () {
```

Comment:
This may be a style thing, but maybe `fun hello() {`

Reply:
fixed it